linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/service_pcscd_enabled/ansible/shared.yml

```diff
@@ -0,0 +1,24 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = enable
+# complexity = low
+# disruption = low
+- name: Enable service pcscd
+  block:
+  - name: Gather the package facts
+    package_facts:
+      manager: auto
+
+  - name: Start service pcscd
+    service:
+      name: "pcscd"
+      state: "started"
+      masked: "no"
+    when:
+    - '"pcsc-lite" in ansible_facts.packages'
+
+  - name: Enable service pcscd
+    ansible.builtin.command:
```

Comment:
If I'm not mistaken we need to add this exception to https://github.com/ComplianceAsCode/content/blob/master/tests/ansible-lint_config.yml because comments are lost during yaml processing by the build system. Let's try it first

Reply:
yep, it doesn't work. We need to add to the global config for ansible lint

Reply:
https://github.com/ComplianceAsCode/content/pull/8312/files#diff-a29c96a0255fd95a3c6b514b2772732e7bac3a6e1cc615f43d0781fb3c4415a4R25